packages/cli/src/commands/init/init.js

```diff
@@ -104,6 +107,87 @@ async function createFromTemplate({
   }
 }
 
+async function installPods(loader: typeof Ora) {
+  process.chdir('ios');
+
+  const hasPods = await fs.pathExists('Podfile');
+
+  if (!hasPods) {
+    return;
+  }
+
+  try {
+    await commandExists('pod');
+  } catch (err) {
+    loader.succeed();
+
+    const {shouldInstallCocoaPods} = await inquirer.prompt([
+      {
+        type: 'confirm',
+        name: 'shouldInstallCocoaPods',
+        message: 'CocoaPods is not installed, do you want to install it?',
+      },
+    ]);
+
+    if (shouldInstallCocoaPods) {
+      try {
+        // First attempt to install `cocoapods`
+        await execa('gem', ['install', 'cocoapods'], {
+          stdio: 'pipe',
+        });
+      } catch (err) {
+        try {
+          // If that doesn't work then try with sudo
+          await execa('sudo', ['gem', 'install', 'cocoapods'], {
+            stdio: 'pipe',
+          });
+        } catch (err) {
+          throw new CLIError(
+            'Error occurred while trying to install CocoaPods, please run this command again.',
+            err,
+          );
+        }
+      }
+
+      loader.start('Installing pods');
+    }
+  }
+
+  try {
+    await execa('pod', ['install'], {
+      stdio: 'pipe',
+    });
+  } catch (err) {
+    throw new CLIError(
+      'Failed to run "pod install", please try to run it manually.',
+      err,
```

Comment:
while testing locally I noticed that when `pod install` fails it doesn't output anything to the console and the error is pretty unhelpful (imagine being a beginner: what's "pod install", why is it running, where do I run it?). How about:

```
Failed to install CocoaPods  dependencies for iOS project, which is required by this template. 
Please try again manually: "cd ./${ProjectName}/ios && pod install".
CocoaPods documentation: ${chalk.dim.underline("https://cocoapods.org/")}
```

The fact that `pod` outputs failures to stdout is not helpful as well. I think we should show the `console.log(err.stderr || err.stdout)` straight away before throwing this.

Reply:
> The fact that pod outputs failures to stdout is not helpful as well.

Do you mean this when you run on `--verbose`? If you run on with it enabled don't you already have the output of errors on `pod install`?

Reply:
It shows up with verbose, but really, would you like to run whole `init` again with `--verbose` flag just to see the error? It's a bad idea. I'm leaning towards the `CLIError` to show the error stack by default if it's there (not related to this PR though)

Reply:
That's a good point, I'll try the `console.log` you suggested, I thought that without `--verbose` they were already suppressed.